### ISSUE TITLE
[a11y] Top Menu for sapo.pt shifted to the right with system font increased

### ISSUE THREAD

**Issue by hteles** [mozilla-mobile/fenix#12782]
Hello Team.

The top menu for https://www.sapo.pt/noticias/atualidade is misaligned in Firefox beta with:

79.0.0-beta.8 (Build #2015752515)
AC: 48.0.4, 7fb662576
GV: 79.0-20200713203149
AS: 61.0.7

The same page renders correctly in Chrome Browser.

Regards.

Chrome rendering
![Screenshot](https://user-images.githubusercontent.com/1479047/88065903-14d72180-cb65-11ea-87fb-81d8bd6b4d9c.jpg)UNITO-UNDERSCORE!20200721-150223!

Trimmed menu on Firefox
![Screenshot](https://user-images.githubusercontent.com/1479047/88065908-16084e80-cb65-11ea-879d-daa90687d834.jpg)UNITO-UNDERSCORE!20200721-150233!

**Comment by mcarare** [comment]
@hteles Thank you for your report! I cannot reproduce this in 79.0.0-beta.9. Can you retest this after you update? Thank you!

**Comment by hteles** [comment]
Hi @mcarare, sorry... i think i found the culprit!

So, as my eyes are getting older, i usually configure firefox -> Acessibility -> Automatic font sizing to True.
Can you please enable Automatic font Sizing and check again? 

Sorry, should have checked this before.

Regards.

**Comment by mcarare** [comment]
@hteles Thank you for the details provided. Actually, I can reproduce this with setting Automating font sizing to false and setting font size to 110% or higher.

**Comment by hteles** [comment]
Hi @mcarare 

Thanks. Probably there's something wrong with the site formating.
Indeed as my eyes get older I need to adjust don't size on Android settings.

**Comment by eliserichards** [comment]
Thanks for your quick triage @mcarare and @hteles!

**Comment by hteles** [comment]
This issue is still a problem with Fenix 86.1.1.
The top menu with News categories is Shifted to the right

**How to reproduce:**

* go to  [](https://www.sapo.pt) 
 - If you set **Automating font sizing to false** and setting **font size to 110% or higher.**
 -If you set larger than default font size on you Android Settings -> Display -> Font size


This does not happen with increasing font size in other browsers like Chrome, Opera, but they don't actually increase the menu text size, like Fenix does. They only increase the Newa Articles size below.

![image](https://user-images.githubusercontent.com/1479047/109144939-bd7fdc00-7759-11eb-812d-d39b9d08f09d.png)

**Comment by mariansima** [comment]
This issue was reproduced on Samsung A8 SM-A530F Android 9 Firefox 93.0a1 2021-08-29

**Comment by sflorean** [comment]
This issue is still reproducible on Nightly 100.0a1 with Samsung A32 (Android 11).

**Comment by hteles** [comment]
I was gonna this go unsolved.
Maybe some day some dev will pick it up.

**Comment by hteles** [comment]
Hi.
Its fixed on 99.1.0 (Build #2015871515),

Thanks
